facedancer/proxy.py

```diff
@@ -348,68 +340,45 @@ def _get_libusb_context(cls):
 
         return cls.context
 
+    @classmethod
+    def _handle(cls) -> usb1.USBDeviceHandle:
+        assert cls.device_handle is not None
+        return cls.device_handle
 
     @classmethod
     def _destroy_libusb_context(cls):
-        """ Destroys our libusb context on closing our python instance. """
+        """Destroys our libusb context on closing our python instance."""
         if cls.device_handle is not None:
-            device = cls.device_handle.getDevice()
-            number = cls.device_handle.getConfiguration()
-            active_configuration = next(filter(lambda c: c.getConfigurationValue() == number, device), None)
-            if active_configuration:
-                for interface in active_configuration:
-                    number = interface[0].getNumber()
-                    try:
-                        cls.device_handle.releaseInterface(number)
-                    except usb1.USBErrorNotFound as e:
-                        log.warning(f"Failed to releace interface {0} for {device}")
-                        pass
-
+            cls._release()
             cls.device_handle.close()
```

Comment:
Can you please revert this refactor. When things go wrong it's better to have the code causing the problem at the same site where the problem occurred!